R/foreign-keys.R

```diff
@@ -1,5 +1,13 @@
 #' Add a reference from one table of a [`dm`] to another
 #'
+#' @description `dm_add_fk()` marks the specified column as the foreign key of table `table` with
+#' respect to the primary key of table `ref_table`.
+#' If `check == TRUE`, then it will first check if the values in column `column` are a subset
+#' of the values of the primary key in table `ref_table`.
+#'
+#' FYI: Mind, that in one of the upcoming updates, compound keys will be supported, and
+#' therefore the syntax of this function will slightly change.
+#'
 #' @inheritParams dm_add_pk
 #' @param column The column of `table` which is to become the foreign key column and
```

Comment:
Here and everywhere?

```suggestion
#' @param columns The column of `table` which is to become the foreign key column and
```